feature-libs/checkout/base/components/checkout-shipping-address/checkout-shipping-address.component.ts

```diff
@@ -142,17 +174,42 @@ export class CheckoutShippingAddressComponent implements OnInit {
   }
 
   selectAddress(address: Address): void {
-    this.checkoutDeliveryAddressFacade.setDeliveryAddress(address);
+    this.busy$.next(true);
+    this.checkoutDeliveryAddressFacade
+      .setDeliveryAddress(address)
+      .pipe(switchMap(() => this.prepareDeliveryModes$))
```

Comment:
Not sure about this use case as we're basically doubling the work on what is already done in the command already and its side effects in the listeners, such as deleting the delivery mode and resetting query and stuff. But in the switchMap stream, we're doing it again.

In conclusion, I don't think we need to touch this stream*

Reply:
Sounds good.